lib/unparser/emitter/meta.rb

```diff
@@ -0,0 +1,8 @@
+module Unparser
+  class Emitter
+    # Namespace class for meta emitters
+    class Meta < self
+      include Terminated
```

Comment:
Instead of the two subclasses you might be able to use handle both nodes here (untested):

```ruby
handle(:__FILE__, :__LINE__)

def dispatch
  emit(node.type)
end
```

Reply:
bump